obs-packaging/wait-obs.sh

```diff
@@ -63,14 +67,45 @@ check_failed(){
 	failed_query=$(osc pr -c  -s  F)
 	regex=".*failed.*"
 	if [[ ${failed_query} =~ ${regex} ]];then
-		printf "%s" "${failed_query}" | column -t -s\;
-		return 1
+		echo "ERROR: Build failed"
+		osc pr -V -s 'F'
+		exit 1
 	fi
+	echo "Nothing failed"
+	osc pr -q -c | tail -n +2 | column -t -s\;
 	return 0
 }
 
+usage() {
+	msg="${1:-}"
+	exit_code=$"${2:-0}"
+	cat <<EOT
+${msg}
+Usage:
+${script_name} [--options]
+
+options:
+	-h, --help: Show this help
+	--no-wait-publish : no wait that OBS publish packages
```

Comment:
rephrasing: `don't wait for OBS to publish packages`

Reply:
agree, thanks for the comments I will send a PR as soon as possible, I wanted to have some CI soon and merge it.